.github/workflows/acceptance.yml

```
@@ -8,11 +8,23 @@ on:
     tags: [v*]
 
 jobs:
-  api:
-    name: API
+  api_batch_1:
+    name: API Chunk 1
```

nit: here and 2 below
```suggestion
    name: API Batch 1
```